tests/e2e/e2e_setup_test.go

```diff
@@ -265,6 +262,37 @@ func (s *IntegrationTestSuite) generateAuthAndBankState(
 		delayedVestingAcc    = c.genesisVestingAccounts[delayedVestingKey]
 	)
 
+	// add jailed validator to staking store
+	pubKey, err := jailedValKey.GetPubKey()
+	s.Require().NoError(err)
+	valAcc, err := jailedValKey.GetAddress()
```

Comment:
```suggestion
	jailedValAcc, err := jailedValKey.GetAddress()
```

since there's jailed and non-jailed vals getting added in the same function would be good to keep a var name that distinguishes them as such. I was confused for a moment below adding the delegations to the genState whether this was actually the jailed validator or not.